src/main/java/com/redhat/jenkins/plugins/bayesian/Bayesian.java

```diff
@@ -46,6 +50,8 @@
 /* package */ class Bayesian {
 
     private static final String DEFAULT_BAYESIAN_URL = "https://recommender.api.openshift.io/";
+    private static final String OSIO_USERS_URL = "https://api.openshift.io/api/users";
```

Comment:
This is good enough for testing purposes, but the URL should be configurable as the value is specific to prod deployment.